src/compiler/diagnosticMessages.json

```diff
@@ -2409,6 +2409,10 @@
         "category": "Error",
         "code": 2729
     },
+    "This export conflicts with the first.": {

```

Comment:
I know the original issue said the message should be `This export conflicts with the first. ` - but that would be if we issued only one diagnostic (on the first `default export`), and then just added a bunch of related spans to it. In this case, we're keeping all the diagnostics, but adding related spans on them. In that case, I think the message should be `The first export default is here.`.

Reply:
cc @DanielRosenwasser since you had the original issue

Reply:
`and here`😂 have we already get this message?